optaplanner-persistence/optaplanner-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <persistence xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              version="2.0"  xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_0.xsd">
-  <persistence-unit name="optaplanner-persistence-jpa-test" transaction-type="JTA">
-    <provider>org.hibernate.ejb.HibernatePersistence</provider>
+  <persistence-unit name="org.optaplanner.persistence.jpa" transaction-type="JTA">
```

Comment:
we'll want to keep the word test in here as it's a test

Reply:
+1